### PR TITLE
EVA-3540 Create new Step for Shutting down accessioning

### PR DESCRIPTION
**Pull request by nitin-ebi**

### CODE REVIEW

File: eva-accession-pipeline/src/test/java/uk/ac/ebi/eva/accession/pipeline/configuration/batch/jobs/CreateSubsnpAccessionsRecoveringStateJobConfigurationTest.java [hunk 1]
```diff
@@ -156,12 +152,18 @@ public void accessionJobShouldRecoverUncommittedAccessions() throws Exception {
...
+        SubmittedVariantEntity entity = new SubmittedVariantEntity(UNCOMMITTED_ACCESSION, "hash-10", model, 1);
+        repository.save(entity);
+
         assertEquals(1, repository.count());
         assertEquals(1, repository.findByAccession(UNCOMMITTED_ACCESSION).size());
         assertEquals(1, blockRepository.count());
 
         // This means that the last committed accession is the previous one to the UNCOMMITTED_ACCESSION
-        assertEquals(UNCOMMITTED_ACCESSION, blockRepository.findAll().iterator().next().getLastCommitted());
+        assertEquals(4999999999L, blockRepository.findAll().iterator().next().getLastCommitted());
```
Comment (apriltuesday) [inline eva-accession-pipeline/src/test/java/uk/ac/ebi/eva/accession/pipeline/configuration/batch/jobs/CreateSubsnpAccessionsRecoveringStateJobConfigurationTest.java:166]:
```suggestion
        assertEquals(UNCOMMITTED_ACCESSION - 1, blockRepository.findAll().iterator().next().getLastCommitted());
```

File: eva-accession-pipeline/src/test/java/uk/ac/ebi/eva/accession/pipeline/configuration/batch/jobs/CreateSubsnpAccessionsRecoverStateTest.java [hunk 2]
```diff
@@ -134,11 +136,11 @@ public void cleanSlate() throws Exception {
         Files.deleteIfExists(Paths.get(inputParameters.getFasta() + ".fai"));
     }
 
-    /**
-     * Note that for this test to work, we prepare the Mongo database in {@link RecoverTestAccessioningConfiguration}.
-     */
     @Test
     public void accessionJobShouldRecoverUncommittedAccessions() throws Exception {
+        // Fill Uncommitted Accessions - accessions that are present in MongoDB but wasn't committed in the block service
+        initialize_mongo_db_with_uncommitted_accessions();
```
Comment (apriltuesday) [inline eva-accession-pipeline/src/test/java/uk/ac/ebi/eva/accession/pipeline/configuration/batch/jobs/CreateSubsnpAccessionsRecoverStateTest.java:142]:
```suggestion
        initializeMongoDbWithUncommittedAccessions();
```

**Review by apriltuesday — APPROVED** [review]

File: eva-accession-pipeline/src/test/java/uk/ac/ebi/eva/accession/pipeline/configuration/batch/jobs/CreateSubsnpAccessionsRecoveringStateJobConfigurationTest.java (hunk 1 above)
Comment (nitin-ebi) [reply eva-accession-pipeline/src/test/java/uk/ac/ebi/eva/accession/pipeline/configuration/batch/jobs/CreateSubsnpAccessionsRecoveringStateJobConfigurationTest.java]: done

File: eva-accession-pipeline/src/test/java/uk/ac/ebi/eva/accession/pipeline/configuration/batch/jobs/CreateSubsnpAccessionsRecoverStateTest.java (hunk 2 above)
Comment (nitin-ebi) [reply eva-accession-pipeline/src/test/java/uk/ac/ebi/eva/accession/pipeline/configuration/batch/jobs/CreateSubsnpAccessionsRecoverStateTest.java]: done

**Pull request merged**